src/CommunityToolkit/Xamarin.CommunityToolkit/ObjectModel/BaseCommand.shared.cs

```diff
@@ -77,12 +77,18 @@ protected int ExecutionCount
 		};
 
 		/// <summary>
-		/// Raises the CanExecuteChanged event.
+		/// Raises the `ICommand.CanExecuteChanged` event.
 		/// </summary>
 		public void RaiseCanExecuteChanged()
 		{
 			// Automatically marshall to the Main Thread to adhere to the way that Xamarin.Forms automatically marshalls binding events to Main Thread
 			Device.BeginInvokeOnMainThread(() => weakEventManager.RaiseEvent(this, EventArgs.Empty, nameof(CanExecuteChanged)));
 		}
+
+		/// <summary>
+		/// Raises the `ICommand.CanExecuteChanged` event.
+		/// </summary>
+		[Obsolete("Use RaiseCanExecuteChanged() instead")]
+		public void ChangeCanExecute() => RaiseCanExecuteChanged();
```

Comment:
Well, we added a new already obsolete method :)

I guess an obsolete attribute should be added to RaiseCanExecuteChanged, right?

Reply:
Hey Andrei! 

I decided to tag it with `[Obsolete]` to reduce the overhead of maintaining and documenting two methods that do the same thing.

I chose to add the `[Obsolete]` tag to `ChangeCanExecute` rather than `RaiseCanExecuteChanged` for two reasons:
- The documentation, samples, and tests already include and leverage `RaiseCanExecuteChanged`
- `RaiseCanExecuteChanged` is a better description of what is actually happening under the hood; it invokes `event ICommand.CanExecuteChanged`

Reply:
Another option is to replace `[Obsolete]` with `[EditorBrowsable(EditorBrowsableState.Never)]`

Reply:
`RaiseCanExecuteChanged` is a better name, but `ChangeCanExecute` provides backwards compatibility with `Command` class (and interface). 
I would also deprecate `RaiseCanExecuteChanged`. If `ChangeCanExecute` would to be renamed (or deprecated and replaces with `RaiseCanExecuteChanged`) this needs to be done in `Command` class too to provide more seamless experience when user needs to switch from `Command` to `AsyncCommand` or backwards.

Reply:
There is no `ChangeCanExecute` in the [`ICommand` interface](https://docs.microsoft.com/dotnet/api/system.windows.input.icommand?view=net-5.0&WT.mc_id=mobile-0000-bramin). `ChangeCanExecute` is just the method name that the creators of Xamarin.Forms came up with; it is not an industry standard.

Adding `ChangeCanExecute` does allow for a seamless transition from `Xamarin.Forms.Command` to `AsyncCommand`.

Reply:
You are current about `ICommand` interface. Need to really start double checking what I'm talking about. This is really confusing method.

But this doesn't change that `Command` has it. Why having method with the same name and functionality in `AsyncCommand` wouldn't provide seamless transition?